old bindgen version crashes build on osx

greetings folks! i think it is a simple problem and I have a fix!

see the error message below.  I isolated the problem to an old `bindgen` version in `libevent-sys`.

updating the bindgen version to 0.69 seems to fix the problem.

(looks like we can use 0.65 instead of 0.69 -- i will update the PR)

(see https://github.com/jmagnuson/libevent-rs/pull/23)

```error-message
error: failed to run custom build command for `libevent-sys v0.2.6`

Caused by:
  process didn't exit successfully: `target/debug/build/libevent-sys-60df0d396610d0bb/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=LIBEVENT_CORE_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rustc-link-search=native=/usr/local/Cellar/libevent/2.1.12_1/lib
  cargo:rustc-link-lib=event_core
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=LIBEVENT_EXTRA_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rustc-link-search=native=/usr/local/Cellar/libevent/2.1.12_1/lib
  cargo:rustc-link-lib=event_extra
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=LIBEVENT_OPENSSL_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rustc-link-search=native=/usr/local/Cellar/libevent/2.1.12_1/lib
  cargo:rustc-link-lib=event_openssl
  cargo:rustc-link-lib=event
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=LIBEVENT_PTHREADS_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rustc-link-search=native=/usr/local/Cellar/libevent/2.1.12_1/lib
  cargo:rustc-link-lib=event_pthreads
  cargo:rustc-link-lib=event
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:include=/usr/local/Cellar/libevent/2.1.12_1/include
  cargo:include=/usr/local/Cellar/openssl@3/3.1.1/include
  cargo:rerun-if-changed=libevent
  cargo:rerun-if-changed=wrapper.h

  --- stderr
  thread 'main' panicked at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/ir/context.rs:861:9:
  "in6_addr_union_(unnamed_at_/Library/Developer/CommandLineTools/SDKs/MacOSX_sdk/usr/include/netinet6/in6_h_153_2)" is not a valid Ident
  stack backtrace:
     0:        0x10442bb13 - std::backtrace_rs::backtrace::libunwind::trace::haf73b04d6a32d84b
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/../../backtrace/src/backtrace/libunwind.rs:116:5
     1:        0x10442bb13 - std::backtrace_rs::backtrace::trace_unsynchronized::hf58a528af87e4be4
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
     2:        0x10442bb13 - std::sys::backtrace::_print_fmt::hf97818f279c655e0
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/sys/backtrace.rs:68:5
     3:        0x10442bb13 - <std::sys::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h4a8027d15e9cc40f
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/sys/backtrace.rs:44:22
     4:        0x10444da2b - core::fmt::rt::Argument::fmt::hca21b106d9911dae
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/core/src/fmt/rt.rs:165:63
     5:        0x10444da2b - core::fmt::write::h8d5eba306d67f99d
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/core/src/fmt/mod.rs:1168:21
     6:        0x10442903e - std::io::Write::write_fmt::h7d261573eb1061e8
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/io/mod.rs:1835:15
     7:        0x10442b901 - std::sys::backtrace::_print::h5718888006d80da7
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/sys/backtrace.rs:47:5
     8:        0x10442b901 - std::sys::backtrace::print::h502b55980befa0c8
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/sys/backtrace.rs:34:9
     9:        0x10442cd39 - std::panicking::default_hook::{{closure}}::hb06be30c74d63c27
    10:        0x10442caba - std::panicking::default_hook::h98a69adb4e304440
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/panicking.rs:292:9
    11:        0x10442d90c - std::panicking::rust_panic_with_hook::h15ffb6f40cb0fdee
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/panicking.rs:796:13
    12:        0x10442d1f5 - std::panicking::begin_panic_handler::{{closure}}::h0ad02b5e0074c52f
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/panicking.rs:670:13
    13:        0x10442bfe9 - std::sys::backtrace::__rust_end_short_backtrace::hd8b7db80cdfcc4a5
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/sys/backtrace.rs:171:18
    14:        0x10442ce0c - rust_begin_unwind
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/panicking.rs:661:5
    15:        0x10446f42a - core::panicking::panic_fmt::hf5ab9b0d4c6b5f74
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/core/src/panicking.rs:74:14
    16:        0x1043ad6bd - proc_macro2::fallback::validate_ident::hd8ae29af5ace6d5a
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/proc-macro2-1.0.85/src/fallback.rs:860:9
    17:        0x1043ad3ce - proc_macro2::fallback::Ident::new_checked::h25ccb791718df424
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/proc-macro2-1.0.85/src/fallback.rs:792:9
    18:        0x1043b0f4f - proc_macro2::imp::Ident::new_checked::hcd7822e7e2aec9a0
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/proc-macro2-1.0.85/src/wrapper.rs:676:50
    19:        0x1043bc744 - proc_macro2::Ident::new::h0d81e833d350face
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/proc-macro2-1.0.85/src/lib.rs:997:21
    20:        0x103f81e5c - bindgen::ir::context::BindgenContext::rust_ident_raw::h6b8df74db726fa29
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/ir/context.rs:861:9
    21:        0x103f81dce - bindgen::ir::context::BindgenContext::rust_ident::hf3ec832d085ae22e
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/ir/context.rs:853:9
    22:        0x103fe373e - <bindgen::ir::comp::CompInfo as bindgen::codegen::CodeGenerator>::codegen::h69276ea46f28bc4b
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:1786:31
    23:        0x103ede076 - <bindgen::ir::ty::Type as bindgen::codegen::CodeGenerator>::codegen::hf1122df804f7a49b
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:814:39
    24:        0x103fc5552 - <bindgen::ir::item::Item as bindgen::codegen::CodeGenerator>::codegen::hb6a0f321fd6fa932
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:500:17
    25:        0x103fe9b54 - <bindgen::ir::comp::CompInfo as bindgen::codegen::CodeGenerator>::codegen::h69276ea46f28bc4b
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:2115:13
    26:        0x103ede076 - <bindgen::ir::ty::Type as bindgen::codegen::CodeGenerator>::codegen::hf1122df804f7a49b
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:814:39
    27:        0x103fc5552 - <bindgen::ir::item::Item as bindgen::codegen::CodeGenerator>::codegen::hb6a0f321fd6fa932
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:500:17
    28:        0x103f9d63c - <bindgen::ir::module::Module as bindgen::codegen::CodeGenerator>::codegen::{{closure}}::h4150667b2f5967a9
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:523:21
    29:        0x103f32be6 - <bindgen::ir::module::Module as bindgen::codegen::CodeGenerator>::codegen::h586b4028e29d6381
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:553:13
    30:        0x103fc5528 - <bindgen::ir::item::Item as bindgen::codegen::CodeGenerator>::codegen::hb6a0f321fd6fa932
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:491:17
    31:        0x103fa95a0 - bindgen::codegen::codegen::{{closure}}::hb68a03e4f818aa26
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:4364:9
    32:        0x103f843e2 - bindgen::ir::context::BindgenContext::gen::h15da651e306457e5
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/ir/context.rs:1173:19
    33:        0x103fa8a94 - bindgen::codegen::codegen::ha0ba9ac0f636147c
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/codegen/mod.rs:4328:5
    34:        0x103f3fdfb - bindgen::Bindings::generate::h828e741249452ec9
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/lib.rs:2438:32
    35:        0x103f3d1a9 - bindgen::Builder::generate::hafc0ffe7b72e47fa
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.60.1/src/lib.rs:1502:9
    36:        0x103e9810c - build_script_build::generate_bindings::h3e5310050f98a639
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libevent-sys-0.2.6/build.rs:191:20
    37:        0x103e984df - build_script_build::main::hc35abb5149eb0794
                                 at /Users/kleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libevent-sys-0.2.6/build.rs:226:5
    38:        0x103e9509e - core::ops::function::FnOnce::call_once::h3c1858581c17ae41
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/core/src/ops/function.rs:250:5
    39:        0x103e99191 - std::sys::backtrace::__rust_begin_short_backtrace::hd3b9297f8dd9fe76
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/sys/backtrace.rs:155:18
    40:        0x103e94db4 - std::rt::lang_start::{{closure}}::h81f9464df23f8888
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/rt.rs:159:18
    41:        0x10442553a - core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once::hd5bae81421f2473b
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/core/src/ops/function.rs:284:13
    42:        0x10442553a - std::panicking::try::do_call::h6c8d7b78a794f34a
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/panicking.rs:553:40
    43:        0x10442553a - std::panicking::try::h4cf403b67666ff1e
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/panicking.rs:517:19
    44:        0x10442553a - std::panic::catch_unwind::h6fa463db1c2fddba
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/panic.rs:350:14
    45:        0x10442553a - std::rt::lang_start_internal::{{closure}}::heb23661a859390a8
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/rt.rs:141:48
    46:        0x10442553a - std::panicking::try::do_call::ha31139b9d340c501
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/panicking.rs:553:40
    47:        0x10442553a - std::panicking::try::hfa4a05c0879be4e2
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/panicking.rs:517:19
    48:        0x10442553a - std::panic::catch_unwind::ha6d4b0139c65d425
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/panic.rs:350:14
    49:        0x10442553a - std::rt::lang_start_internal::h78ca0f5050eb78d3
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/rt.rs:141:20
    50:        0x103e94d87 - std::rt::lang_start::h76f3fdb4e24425d4
                                 at /rustc/8fcd4dd08e2ba3e922d917d819ba0be066bdb005/library/std/src/rt.rs:158:17
    51:        0x103e98528 - _main
    52:     0x7ff80fe69366 - <unknown>
warning: build failed, waiting for other jobs to finish...
make: *** [build] Error 101
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,5 +30,5 @@ verbose_build = [ "libevent-sys/verbose_build" ]
 
 [dependencies]
 bitflags = "1.2"
-libevent-sys = { version = "0.2", path = "libevent-sys", default-features = false }
+libevent-sys = { version = "0.3", path = "libevent-sys", default-features = false }
 

--- a/README.md
+++ b/README.md
@@ -45,7 +45,7 @@ base.run();
 
 ## Minimum Supported Rust Version (MSRV)
 
-This crate is guaranteed to compile on stable Rust 1.57.0 and up. It might compile
+This crate is guaranteed to compile on stable Rust 1.70.0 and up. It might compile
 with older versions but that may change in any new patch release.
 
 ## License

--- a/libevent-sys/Cargo.toml
+++ b/libevent-sys/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "libevent-sys"
-version = "0.2.6"
+version = "0.3.0"
 authors = ["Steven vanZyl <rushsteve1@rushsteve1.us>",
            "Jon Magnuson <jon.magnuson@gmail.com>",
            "Grant Elbert <elbe0046@gmail.com>"]

